api/v1/verticadb_webhook.go

```diff
@@ -1701,6 +1728,199 @@ func (v *VerticaDB) checkSandboxPrimary(allErrs field.ErrorList, oldObj *Vertica
 	return allErrs
 }
 
+// ensures a new subcluster or sandbox to be added to a vdb has Shutdown field set to false
```

Comment:
```suggestion
// validateNewSBoxOrSClusterShutdownUnset ensures a new subcluster or sandbox to be added to a vdb has Shutdown field set to false
```
Follow the same forma for other functions